examples/seq2seq/seq2seq.py

```diff
@@ -217,7 +221,9 @@ def main():
     optimizer = chainer.optimizers.Adam()
     optimizer.setup(model)
 
-    train_iter = chainer.iterators.SerialIterator(train_data, args.batchsize)
+    train_iter = chainer.iterators.SerialIterator(train_data,
+                                                  args.batchsize,
+                                                  shuffle=False)
```

Comment:
At least, adjacent lines in wmt dataset is not independent. Thus, it can cause bias in stochastic mini-batch training, which might be harmful. I would like train iterator to be shuffled in practice.